internal/pkg/chainsync/fetcher/graphsync_fetcher.go

```diff
@@ -436,9 +433,6 @@ func (gsf *GraphSyncFetcher) loadAndVerifyFullBlock(ctx context.Context, key blo
 				return err
 			}
 
-			if err := gsf.validator.ValidateUnsignedMessagesSyntax(ctx, messages); err != nil {
```

Comment:
These have been calling unimplemented stubs for a long time.  I didn't try to plug in the right thing here but can file an issue for following up on this with actually message validation.

Reply:
Thanks, please do. I do believe we need something here else catchup sync messages are going unvalidated.

Can you reference https://github.com/filecoin-project/go-filecoin/issues/3312 here rather than leave no trace?